src/classes/Stringy.ts

```diff
@@ -20,4 +20,22 @@ export default class Stringy {
 
         return refinedPrefix.length === matches.length;
     }
+
+    isSuffix(suffix: string, isCaseSensitive: boolean = false) {
+        let matches = '';
+        const refinedSuffix = isCaseSensitive ? suffix : suffix.toLowerCase();
+        const refinedText = isCaseSensitive ? this.text : this.text.toLowerCase();
+
+        const suffixTextDifference = refinedText.length - refinedSuffix.length;
+
+        for(let index = refinedSuffix.length - 1; index >= 0; index--) {
```

Comment:
Codacy has a fix for the issue: [Replace `········for` with `····for·`](https://app.codacy.com/gh/VictorMaria/Strings-and-Arrays/pullRequest?prid=9807711)
```suggestion
    for (let index = refinedSuffix.length - 1; index >= 0; index--) {
```